contracts/src/BeefyClient.sol

```diff
@@ -447,7 +465,7 @@ contract BeefyClient {
     ) internal view {
         Ticket storage ticket = tickets[ticketID];
         // Verify that enough signature proofs have been supplied
-        uint256 signatureCount = minimumSignatureThreshold(vset.length);
+        uint256 signatureCount = ticket.signatureCountRequired;
```

Comment:
```suggestion
        uint256 numRequiredSignatures = ticket.signatureCountRequired;
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/46250d8ff06a500a194a2c57acd59397e343af46